src/javaRosa/util.js

```diff
@@ -252,7 +252,8 @@ define([
     var insertOutputRef = function(vellum, target, path, mug, dateFormat) {
         var output = getOutputRef(path, dateFormat),
             form = vellum.data.core.form;
-        if ((!mug && vellum.opts().features.rich_text) || (mug && mug.supportsRichText())) {
+        if ((!mug && form.useRichText !== false) ||
```

Comment:
I thought someone had added that to the [form constructor](https://github.com/dimagi/Vellum/blob/bad7a8faa5b58cb27fcf5c2c2230d32c80378481/src/form.js#L111) a while ago. That seems like an ideal place for it.